<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serveurcrawler</class>
 <widget class="QWidget" name="serveurcrawler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>serveurcrawler</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>550</y>
     <width>336</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelPort">
      <property name="text">
       <string>Port d'ecoute</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBoxPort">
      <property name="minimum">
       <number>1024</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
      <property name="value">
       <number>8888</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonLancement">
      <property name="text">
       <string>lancement</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonQuitter">
      <property name="text">
       <string>quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>serveurcrawler</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
